pkgs/test/CHANGELOG.md

```diff
@@ -9,11 +13,11 @@
 
 ## 1.5.0
 
-* Depend on `package:test_core` for core functionality. 
+* Depend on `package:test_core` for core functionality.
```

Comment:
Is this an intended change?

Reply:
Guessing this is @jakemac53 's markdown auto-format on save thingy

Reply:
Yep - figured I might as well leave it in since it cleans it up a bit